src/components/article-components/ArticleComments.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { api } from '../../api';
+import CommentCard from './CommentCard';
+
+const ArticleComments = ({ articleId }) => {
+	const [comments, setComments] = useState([]);
```

Comment:
You might want to add some separate indication that comments are loading for your user, as the loading state in your Article component is not a guarantee that the api request for comments will have completed. But be careful to think about scenarios where an article doesn't have any comments 🤔

Reply:
I didn't realize i was using the same loading state, i must have autopilot it in, of course i'll need a loading state where i actually fetch the comment data